aio_pika/tools.py

```diff
@@ -2,6 +2,38 @@
 from functools import partial
 
 
+def iscoroutinepartial(fn):
```

Comment:
It's looks like an `asyncio.coroutine` decorator. I often use it in similar cases. Can you try apply this instead?

E.g.
```python

def feedback(func, *args, **kwargs):
    loop.create_task(asyncio.coroutine(func)(*args, **kwargs))

```

p.s: I do not insist, you can try to just get rid of it.